### ISSUE TITLE
Token cache conflict

### ISSUE THREAD

**Issue by the-eater** [netauth/netauth#9]
running 

```
sudo netauth auth --entity root inspect-token
netauth auth --entity root inspect-token
```

will result in the following behaviour

```
[eater@tamako netauth]$ sudo netauth auth inspect-token
Secret: 
This token was issued to 'root'
 Capabilities:
  - GLOBAL_ROOT
[eater@tamako netauth]$ netauth auth --entity root inspect-token
Secret: 
Error caching token: open /tmp/root.nt: permission denied
```

**Comment by the-maldridge** [comment]
This is explicitly called out [in the quickstart](https://www.netauth.org/ecosystem/netauth/) but I agree this can be confusing.  Perhaps this should be added to the docs as well?

**Comment by the-eater** [comment]
oh! I missed that part, I didn't even know that quickstart part was there hm.

but my problem is more with writing the token to the cache e.g. the process is now

```
try read token from cache ─ success ───────────────────────────────> Do Thing
└ fail ──> authenticate entity ──> write new received token to cache ──>┘
```

if writing the new token to cache fails for _any_ reason netauth will quit, even tho it has all the info to continue for this current session

this means that if anyone authenticated as user X, _nobody else_ until /tmp is wiped or the cache file is deleted can authenticate as user X, even though they know the secret and are able to generate a new token

**Comment by the-maldridge** [comment]
Resolved by 25192fd.  Cache write failure is no longer a fatal error.
